azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -264,6 +266,8 @@ def create_from_connection_string(cls, connection_string, **kwargs):
         :param bool connection_retry: Attempt to re-establish a dropped connection (Default: True)
         :param int connection_retry_interval: Interval, in seconds, between attempts to
             re-establish a dropped connection (Default: 10)
+        :param bool ensure_desired_properties: A boolean to determine whether or not to refresh
```

Comment:
Redundant, type is already listed in the param. Also needs more information. Should instead be

"Ensure the most recent desired properties patch has been received upon reconnections (Default: True)"

Here and everywhere.

Reply:
Fixed in all cases, thanks